addons/im_livechat/static/tests/embed/livechat_service.test.js

```diff
@@ -122,7 +122,7 @@ test("Only necessary requests are made when creating a new chat", async () => {
                 temporary_id: 0.81,
             },
             post_data: {
-                body: "Hello!",
+                body: "<odoo-markdown>Hello!</odoo-markdown>",
```

Comment:
AL suggested to use `<pre>` tag with `data-` to allow this new format to be fully "compatible" with HTML in case the lib is not available, so that it just renders as a markdown code block in standard HTML.

Reply:
I see, I would argue the following:

* In HTML5 we can create whatever markup we want, it doesn't make it incompatible with anything I know of.
* For the people with markdown working, it will be invalid HTML since `<pre>` doesn't accept a lot of elements as children:
![image](https://github.com/user-attachments/assets/0b387819-d709-40f1-8c20-d6fbf6acdc99)
When using a custom element: 
![image](https://github.com/user-attachments/assets/a7c28b5f-a6d3-4823-8aaf-74f4bbcc4a64)